.pre-commit-config.yaml

```diff
@@ -13,3 +13,13 @@ repos:
         entry: tflint --recursive --fix
         language: system
         pass_filenames: false
+  - repo: local
+    hooks:
+      - id: trufflehog
+        name: TruffleHog
+        description: Detect secrets in your data.
+        entry: bash -c 'trufflehog git file://. --since-commit HEAD --no-verification  --fail'
+        # For running trufflehog in docker, use the following entry instead:
+        # entry: bash -c 'docker run --rm -v "$(pwd):/workdir" -i --rm trufflesecurity/trufflehog:latest git file:///workdir --since-commit HEAD --only-verified --fail'
```

Comment:
This comment would probabably be better in the README. And also probably better to show how to wrap that docker command as an alias or script on the path rather than having people hacking this file.